backend/src/main/java/com/greenfoxacademy/backend/controller/UserController.java

```diff
@@ -63,6 +64,20 @@ public Map<String, String> handleValidationExceptions(MethodArgumentNotValidExce
     });
     return errors;
   }
-  
+
+  /**
+   * This method login an existing user.
+   *
+   * @param loginUserDto the user to be logged in
+   * @return a response entity
+   */
+  @CrossOrigin(origins = "http://localhost:5173")
+  @PostMapping("/login")
+  public ResponseEntity<?> loginUser(@RequestBody LoginUserDto loginUserDto) {
+    Map<String, String> response = new HashMap<>();
+    response.put("Response", "Login OK");
+    return ResponseEntity.ok().body(response);
+  }
```

Comment:
This is looking good, but probably will conflict with the other PR. Let's merge one of these and resolve the conflict in a way to pass the tests. 💪 
But this is looking good for just mocking things.

Reply:
One more thing: I do not mind building a JSON-object by ourselves, but since JAva is ... you know a language with classes and object 😄 you can create a response DTO to work with this endpoint.